vcpkg.json

```diff
@@ -36,6 +36,16 @@
       },
       "zlib"
     ],
+    "overrides": [
+      {
+        "name": "openssl",
+        "version": "3.0.8#2"
+      },
+      {
+        "name": "physx",
+        "version": "4.1.2#6"
+      }
```

Comment:
It might be nice to add a `$comment` to indicate why we have these overrides. I don't know if this is standard JSON, but vcpkg.json allows any entry beginning with a $ to serve as a comment, so:
```json
      {
        "name": "physx",
        "version": "4.1.2#6",
        "$comment": "This is the last version of PhysX 4.1.2 in vcpkg. PhysX 5 drops support for XYZ."
      }
```